package.json

```diff
@@ -27,7 +27,7 @@
     ]
   },
   "peerDependencies": {
-    "typeorm": "^0.2.7"
+    "typeorm": "~0.3.0"
```

Comment:
What's the reason for using `~` here rather than `^`? Wouldn't this make it incompatible with `>=0.4.0`?

Reply:
Yeah. That was intentional. `0.3.x` has some breaking syntax fixed in this commit https://github.com/Mrtenz/typeorm-store/pull/21/commits/b068dd03eafaf22f87d9e80044d41d163437e02b. Just in case`0.4.x` introduce breaking change?